### PR TITLE
Api & Form/List Cleanup/Improvements

### PR DESCRIPTION
**Pull request by MTTParry**
- When a debt is added, **I** get an email (this will be changed later). This is API progress.
- BillList & DebtList only show dates now, not times
- debtForm & billForm now have textareas for the notes' inputs
- tip_rate takes an integer (the db has been updated) and the helpers take this into account
- billForm POST has been fixed (see above)

### CODE REVIEW

File: client/src/components/Bills/BillForm.js [hunk 1]
```diff
@@ -125,9 +131,9 @@ const BillForm = (props) => {
         />
         %
         <br />
-        Tip Amount: {(bill.subtotal * bill.tip_rate) / 100}
+        Tip Amount: ${GetTipAmount(bill)}
```
Comment (sfurlani) [inline client/src/components/Bills/BillForm.js]: ❗ if it was calculated in the `setState` above, you can just use `tempBill.tip_total` here?

File: client/src/components/helper/FindTaxRate.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export function GetTaxRate(bill) {
+  let getTaxRate = (bill.tax_total / bill.subtotal);
```
Comment (sfurlani) [inline client/src/components/helper/FindTaxRate.js]: ❗ should prevent divide by 0 errors

`if (bill.subtotal === 0) { return 0; }` or something

**Review by sfurlani — APPROVED** [review]
A few comments but can be fixed in the next round of work

File: client/src/components/helper/FindTaxRate.js (hunk 2 above)
Comment (MTTParry) [reply client/src/components/helper/FindTaxRate.js]: Sounds good. Added it.

File: client/src/components/Bills/BillForm.js (hunk 1 above)
Comment (MTTParry) [reply client/src/components/Bills/BillForm.js]: GetTipAmount takes the `tip_rate` and the `subtotal`.

If you look at line 28, `tempBill.tip_total = GetTipAmount(tempBill)`

Is that what you mean, or did you want me to do `{GetTipAmount(tempBill)}` on line 134?

Comment (MTTParry) [reply client/src/components/Bills/BillForm.js]: Changed it to `Tip Amount: ${bill.tip_total}` so it doesn't have to do as much math.

**Pull request merged**